packages/aws-cdk-lib/aws-events-targets/lib/redshift-query.ts

```diff
@@ -0,0 +1,134 @@
+import { bindBaseTargetConfig, singletonEventRole } from './util';
+import * as events from '../../aws-events';
+import * as iam from '../../aws-iam';
+import * as secretsmanager from '../../aws-secretsmanager';
+import * as sqs from '../../aws-sqs';
+
+/**
+ * Configuration properties of an Amazon Redshift Query event.
+ */
+export interface RedshiftQueryProps {
+
+  /**
+   * The Amazon Redshift database to run the query against.
+   *
+   * @default - dev
```

Comment:
Why have a default for the database?

Reply:
The cluster and namespace resources deploy a default `dev` database when one is not specified. Therefore, I picked `dev` by default. If you want however, I can remove the default assignment if you think that's best?

Reply:
Can you point me to where this is? I can't find where this is happening from.

Reply:
My bad, I was wrong. In the serverless module, there is no mention of the default name for the redshift database, and the cluster module uses the name `default_db`, not `dev`. That is my mistake. I will work on removing the default assignment of the database name

Reply:
This has now been changed